packages/vscode-extension/src/commandsTreeViewProvider.ts

```diff
@@ -373,7 +373,7 @@ export class TreeViewCommand extends vscode.TreeItem {
       this.command = {
         title: label,
         command: commandId,
-        arguments: [CommandsTreeViewProvider.TreeViewFlag]
+        arguments: [[CommandsTreeViewProvider.TreeViewFlag]]
```

Comment:
I need second parameter to mark if showDialog when sign in

Reply:
I see